### PR TITLE
Remove `PatientTeleConsultation` model

### LINKED ISSUE #2178: Remove `PatientTeleConsultation` model
Remove all references to `PatientTeleConsultation` model as tele_consultation_history is not used anymore

### PR DESCRIPTION
**Pull request by jasonmokk**
## Proposed Changes

This PR removes all references to the unused `PatientTeleConsultation` model and `tele_consultation_history`.
### Associated Issue

Fixes #2178 

## Merge Checklist

- [ ] Tests added/fixed
- [ ] Update docs in `/docs`
- [x] Linting Complete
- [ ] Any other necessary step

_*Only PR's with test cases included and passing lint and test pipelines will be reviewed*_

@coronasafe/care-backend-maintainers @coronasafe/care-backend-admins

### CODE REVIEW

**Review by sainak — APPROVED** [review]

**Review by rithviknishad — APPROVED** [review]

File: care/facility/models/patient.py [hunk 1]
```diff
@@ -431,9 +431,6 @@ def __str__(self):
         return f"{self.name} - {self.year_of_birth} - {self.get_gender_display()}"
 
     @property
```
Comment (praffq) [inline care/facility/models/patient.py]: @jasonmokk  you should probably remove `@property` decorator as well

Comment (jasonmokk) [reply care/facility/models/patient.py]: Got it, thanks for the review. PR has been updated.

**Comment by praffq** [comment]
@jasonmokk  also run `make test` command once and see if test cases are passing or not , if not then fix those

**Comment by sainak** [comment]
@vigneshhari this is good for merge

**Review by vigneshhari — APPROVED** [review]

**Pull request merged**